code/datums/traits/negative.dm

```diff
@@ -736,6 +736,15 @@
 	if(H.stat == DEAD)
 		return
 
+	if(H.reagents.has_reagent(/datum/reagent/medicine/epinephrine))
+		return
+	if(H.reagents.has_reagent(/datum/reagent/medicine/atropine))
+		return
+	if(H.reagents.has_reagent(/datum/reagent/medicine/diphenhydramine))
+		return
+	if(H.reagents.has_reagent(/datum/reagent/medicine/synaphydramine))
+		return
+
```

Comment:
why not just make a list of reagents and loop through the list checking if they have one of them?

Reply:
seems overengineered